plugin/pkg/admission/serviceplan/defaultserviceplan/admission_test.go

```diff
@@ -558,6 +558,41 @@ func TestWithNoPlanSucceedsWithMultiplePlansFromDifferentClasses(t *testing.T) {
 	}
 }
 
+func TestWithUPSWorks(t *testing.T) {
```

Comment:
Is this just testing to make sure that the admission controller doesn't mess with the UPS flag?

Reply:
If so, shouldn't we have 2 tests? one with userProvided set to true and one set to false - and then line 579 have tc.userProvided on the right side of the expression?

Reply:
To test that it doesn't mess with it in both situations